### PR TITLE
Fix issue of SNSPowderReduction in case that can run is for vanadium

### LINKED ISSUE #15301: SNSPowderReduction error
[SNSPowderReduction.py#L1129](https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py#L1129) throws an error in NOMAD auto-reduction

### PR DESCRIPTION
**Pull request by wdzhou**
Fixes #15301.

### CODE REVIEW

**Comment by peterfpeterson** [comment]
The builds broke in a weird way. I just resubmitted them.

**Comment by wdzhou** [comment]
This is the error: 
```
00:17:36 /home/builder/Jenkins/workspace/pull_requests-rhel6/build/MantidQt/MantidWidgets/ui_ColorBarWidget.h:24:41: fatal error: MantidQtAPI/QScienceSpinBox.h: No such file or directory
00:17:36  #include "MantidQtAPI/QScienceSpinBox.h"
00:17:36                                          ^
00:17:36 compilation terminated.
```
Jose added this to master.  Then it is left in build directory.   I met the same issue this afternoon.  'make clean' won't remove this.  Finally I had to delete all the files in my build directory.

**Comment by peterfpeterson** [comment]
Do you know about --allow-empty?

**Comment by wdzhou** [comment]
nope.  first time heard of --allow-empty.

**Comment by peterfpeterson** [comment]
You forgot to remove the print debugging

File: Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py [hunk 1]
```diff
@@ -679,19 +683,22 @@ def _loadAndSum(self, run_number_list, outName, **filterWall):
...
-        # Rename to user specified output workspace
-        if sample_ws_name != outName:
-            api.RenameWorkspace(InputWorkspace=sample_ws_name,
-                                OutputWorkspace=outName)
-
+        # Normalize by current with new name
         if self._normalisebycurrent is True:
             self.log().warning('[SPECIAL DB] Normalize current to workspace %s' % sample_ws_name)
-            temp_ws = self.get_workspace(sample_ws_name)
+            # temp_ws = self.get_workspace(sample_ws_name)
```
Comment (peterfpeterson) [inline Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py]: Should this be commented out?

**Pull request merged**